Lib/test/test_typing.py

```diff
@@ -894,6 +906,93 @@ class C(Generic[Unpack[Ts]]): pass
         self.assertNotEqual(C[Unpack[Ts1]], C[Unpack[Ts2]])
 
 
+class TypeVarTuplePicklingTests(BaseTestCase):
+    # These are slightly awkward tests to run, because TypeVarTuples (and things
+    # made generic using TypeVarTuples) are only picklable if defined in the
```

Comment:
The parenthetical isn't really accurate; I suggest removing it.

Reply:
Done.